internal_metrics/dimensions.yaml

```diff
@@ -0,0 +1,171 @@
+az:
+  brief: availability zone
+  description: availability zone
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.clock_offset_ns, ebpf_net.codetiming_avg_ns, ebpf_net.collector_health
```

Comment:
IDK about listing each associated metric in this way.  I know I suggested something like this, but now that I see it it looks too cluttered and possibly more confusing than just leaving these out.  I think for internal metrics less detail may be more, especially since they may change.  Maybe we should only provide detailed docs of internal metrics that we expect the support team to be able to take action on, and the rest that require dev intervention also require reading the source?  I don't really know the answer, might be worth getting the opinions of others.

Reply:
OK, I'm fine with it if it is what Jonah/the team/support wants/needs, it just seems kinds busy :^)